docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -40,20 +45,19 @@ image::images/open-telemetry-elastic-arch.png[OpenTelemetry Elastic architecture
 [[open-telemetry-elastic-works]]
 ==== How the OpenTelemetry Collector works
 
-The OpenTelemetry collector uses three different types of components to handle data: `receivers`, `processors`, and `exporters`.
+The OpenTelemetry collector uses the following three types of components to handle data:
 
 * `receivers`: Configure how data gets to the collector. At least one receiver must be configured.
-* `processors`: Defines optional transformations that occurs between receiving and exporting data.
-* `exporters`: Configures how data is sent to its destination--in this case, the Elastic Stack.
+* `processors`: Defines optional transformations that occur between receiving and exporting data.
+* `exporters`: Configures how data is sent to its destination--in this case, the {stack}.
 
-Once a `receiver`, `processor`, and `exporter` is defined, `pipelines` can be configured in the `services` section of your configuration. Specifically, a `traces` pipeline will define the path of trace data through your collector, and bring all three of these components together.
+Once a `receiver`, `processor`, and `exporter` is defined, `pipelines` can be configured in the `services` section of your configuration.
+The `traces` and `metrics` pipelines define the path of trace data and metrics through your collector and bring all three of these components together.
 
-TIP: More information is available in the
-{ot-pipelines}[OpenTelemetry pipeline docs]
+TIP: More information is available in the {ot-pipelines}[OpenTelemetry pipeline docs].
```

Comment:
OTel got rid of their pipeline docs and this link is now a 404. We should update it on line six. Maybe this? https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/README.md

Reply:
or https://opentelemetry.io/docs/collector/configuration/ ?

Reply:
Thanks. I added https://opentelemetry.io/docs/collector/configuration/